src/coreclr/debug/daccess/request.cpp

```diff
@@ -363,7 +340,7 @@ ClrDataAccess::GetThreadpoolData(struct DacpThreadpoolData *threadpoolData)
 
     threadpoolData->AsyncTimerCallbackCompletionFPtr = (CLRDATA_ADDRESS) GFN_TADDR(ThreadpoolMgr__AsyncTimerCallbackCompletion);
     SOSDacLeave();
-    return hr;
+    return E_NOTIMPL;
```

Comment:
I meant the whole method can be just:
```
HRESULT
ClrDataAccess::GetThreadpoolData(struct DacpThreadpoolData *threadpoolData)
{
    return E_NOTIMPL;
}
```

Reply:
Is the part of an official API? If yes, we should mark this PR as a `breaking-change`.

Reply:
This is not an official documented API.